priority_queue.py

```diff
@@ -0,0 +1,47 @@
+from queue import Queue
+
+
+class Priority_Queue(object):
+
+    def __init__(self):
+        self.high = Queue()
+        self.low = Queue()
+
+    def insert(self, item, priority="high"):
+        """
+        Insert a queue first by priority, then by first-in-last-out.
+
+        Priority_Queue does not track the absolute order that all items
+        were inserted into the queue, and only provides Priority_Queue
+        behavior through pop() and insert() functions.
+        Priority levels are abritrarily set to only 'high' or 'low'.
+        """
+        if priority == "high":
+            self.high.enqueue(item)
+        elif priority == "low":
+            self.low.enqueue(item)
+        else:
+            raise ValueError("Priorty must be literal 'high' or 'low'.")
```

Comment:
this is pretty restrictive.  If I decide as a customer that I need to have a five-level priority system, you have to completely rewrite the entire class to handle that.  Can you think of a way to make this more flexible without making it more complex?